test/server-unit/xlsx.js

```diff
@@ -0,0 +1,33 @@
+const xlsx = require('../../server/lib/renderers/xlsx');
+
+const { expect } = require('chai');
+const _ = require('lodash');
+
+describe('util.js', () => {
+
+  it('Shoul return a xlsx buffer', (done) => {
+    const data = {
+      rows : [{ Firstname : 'Alice', Lastname : 'Bob' }],
+    };
+    xlsx.render(data)
+      .then(reportStream => {
+        expect(_.isBuffer(reportStream)).to.be.equal(true);
+        done();
+      })
+      .catch((error) => {
+        done(error);
+      });
+  });
+
+  it('Shoul works for an empty object', (done) => {
+    const data = {};
+    xlsx.render(data)
+      .then(reportStream => {
+        expect(_.isBuffer(reportStream)).to.be.equal(true);
```

Comment:
Just FYI, I think you can test for a buffer using [the `a()` method](http://www.chaijs.com/api/bdd/#atype-msg):
```js
expect(reportStream).to.be.a('buffer');
```